### PR TITLE
Fix/convert encounter to dex

### LINKED ISSUE #117: Wrong sprites- discord webhook posts
**Describe the bug**
I use the discord webhook feature. Since the major update, sprites have been consistently wrong on the discord posts that pop up when a match is found. 
Everything else is working normally, and the sprite I see on raid crawler is correct. 

**Screenshots**
Screenshot 2023-01-24 091040

### PR DESCRIPTION
**Pull request by lfmundim**
Manual table of conversions for encounter vs dex number. Dex number order taken from serebii, encounter taken from pkhex.

above is before, below is now

image

fixes #117

### CODE REVIEW

File: Structures/NotificationHandler.cs [hunk 1]
```diff
@@ -77,13 +78,14 @@ public static object GenerateWebhook(Config c, ITeraRaid encounter, Raid raid, I
...
             var ivs = IVsStringEmoji(c, ToSpeedLast(blank.IVs), c.IVsStyle, c.IVsSpacer, c.VerboseIVs, emoji);
-            var sprite_name = SpriteName.GetResourceStringSprite(blank.Species, blank.Form, blank.Gender, blank.FormArgument, blank.Generation, Raid.CheckIsShiny(raid, encounter));
+            var sprite_name = SpriteName.GetResourceStringSprite(dexNumber, blank.Form, blank.Gender, blank.FormArgument, blank.Generation, Raid.CheckIsShiny(raid, encounter));
             var moves = new ushort[4] { encounter.Move1, encounter.Move2, encounter.Move3, encounter.Move4 };
             var movestr = string.Concat(moves.Where(z => z != 0).Select(z => $"{Raid.strings.Move[z]}ㅤ\n")).Trim();
             var extramoves = string.Concat(encounter.ExtraMoves.Where(z => z != 0).Select(z => $"{Raid.strings.Move[z]}ㅤ\n")).Trim();
             var area = $"{Areas.Area[raid.Area - 1]}" + (c.ToggleDen ? $" [Den {raid.Den}]ㅤ" : "ㅤ");
             var instance = " " + c.InstanceName;
             var rewards = GetRewards(c, RewardsList, emoji);
+
```
Comment (jnovack) [inline Structures/NotificationHandler.cs:88]: By far the most important line in this commit. :)

Comment (jnovack) [reply Structures/NotificationHandler.cs:88]: No, no. I prefer the readability.

**Comment by pifopi** [comment]
I might be wrong but I think updating pkhex core to latest version would fix the issue as well ?

**Comment by LegoFigure11** [comment]
Correct, and that's the approach we will be going with

**Comment by lfmundim** [comment]
Yeah the pr was up before pkhex updated, I'll close it

**Pull request closed without merging**